docker-compose.yaml

```diff
@@ -44,7 +44,7 @@ services:
             - NANGO_LOGS_ES_PWD
             - FLAG_SERVE_CONNECT_UI=${FLAG_SERVE_CONNECT_UI:-true}
         volumes:
-            - './packages/shared/providers.yaml:/usr/nango-server/src/packages/shared/providers.yaml'
+            - './packages/providers/providers.yaml:/usr/nango-server/src/packages/providers/providers.yaml'
```

Comment:
This needs a breaking changes notice (I don't think a lot of people use this but this)